### PR TITLE
Bring data usage up to GFS v16.3.12

### PR DESCRIPTION
**Pull request by ADCollard**
Modifications to `global_anavinfo.l127.txt`, `global_convinfo.txt`, and `global_satinfo.txt` to turn on Himawari-8 and -9 clear sky radiances; GOES-18 clear sky radiances; NOAA-21 CrIS and ATMS radiances; and Metop-C scatterometer winds.

This includes the relevant changes to [GSI branch release/gfsda.v16](https://github.com/NOAA-EMC/GSI/tree/release/gfsda.v16) from the following commits.

The new file `Rcov_crisn21` is a binary and will need to be added separately.

```
commit 681e2ee32a637931540786c0b5ca04e81e971a60 (HEAD -> release/gfsda.v16, tag: gfsda.v16.3.12, origin/release/gfsda.v16)
Author: Andrew Collard <40322596+ADCollard@users.noreply.github.com>
Date:   Mon Dec 4 11:54:11 2023 -0500

    Add activation of Himawari-9 to the implementation branch (for the gfsda.v16.3.12 tag) (#664)

commit e593e20ee29df275ffe56b82ad4810e9104be43d
Author: Andrew Collard <40322596+ADCollard@users.noreply.github.com>
Date:   Wed Nov 29 13:36:46 2023 -0500

    Turn on NOAA-21, GOES-18 and Metop-C ASCAT in Release/gfsda.v16 (#660)

commit 7fc1a3d2a01bb5730accf3bf4dc3367c2a0bba5e (tag: gfsda.v16.3.10)
Author: Andrew Collard <40322596+ADCollard@users.noreply.github.com>
Date:   Fri Oct 20 05:56:26 2023 -0400

    Correct typo in exglobal_atmos_analysis.sh (#644)

commit 3fc1ce6f141b380a7ef4f5def0880033c5fca846
Author: Andrew Collard <40322596+ADCollard@users.noreply.github.com>
Date:   Thu Oct 5 10:07:03 2023 -0400

    Changes to add NOAA-21 and GOES-18 radiances in monitor mode (#636)
    
    Turn on NOAA-21 and GOES-19 radiances in operations in monitoring mode.
    
    scripts/exglobal_atmos_analysis.sh: Add processing of GOES-18 CSRs,
    NOAA-21 ATMS and CrIS, remove HIRS
    fix/global_anavinfo.l127.txt: Point to new file, Rcov_crisn21
    fix/global_satinfo.txt: Down weighted NOAA-21 CrIS radiances relative to
    NOAA-20
    fix/Rcov_crisn21: New file
    
    The fix branch is DA_GFSv16.3.9_global_only
```

### CODE REVIEW

**Comment by ADCollard** [comment]
I cannot assign people to this PR, so I am explicitly tagging people here:

@KateFriedman-NOAA @RussTreadon-NOAA @emilyhcliu @HaixiaLiu-NOAA 

(@ClaraDraper-NOAA for information)

**Review by KateFriedman-NOAA — DISMISSED** [review]
From my understanding, the changes look to be what I expect for turning on assimilation of NOAA21 (e.g. `-1` to `1` switch updates). Beyond that, I will defer to DA developer reviews for the correctness of the changes. I approve but am not the final say.

File: global_anavinfo.l127.txt
Comment (RussTreadon-NOAA) [inline global_anavinfo.l127.txt:72]: The operational `global_anavinfo.l127.txt` does not contain the `state_vector_efsoi` section but do we want to retain this section in `develop`?

**Review by RussTreadon-NOAA — COMMENTED** [review]
Looks good.  Only one question about `state_vector_efsoi` section in `global_anavinfo.l127.txt`

**Review by emilyhcliu — APPROVED** [review]

File: global_anavinfo.l127.txt
Comment (ADCollard) [reply global_anavinfo.l127.txt:72]: @RussTreadon-NOAA Yes, I was not planning to remove it.   Hopefully we can use efsoi in the future!

Comment (RussTreadon-NOAA) [reply global_anavinfo.l127.txt:72]: Agreed.  Would you please add the efsoi section back into your copy of [`global_anavinfo.l127.txt`](https://github.com/ADCollard/GSI-fix/blob/develop/global_anavinfo.l127.txt)?

**Review by ClaraDraper-NOAA — APPROVED** [review]
I can only review this in terms of impact on my soil analysis changes, and there are no issues on that front.

**Comment by RussTreadon-NOAA** [comment]
@ADCollard , would you please add 
```
 state_vector_efsoi::
 !var     level  itracer source     funcof
  u       127      0     met_guess    u
  v       127      0     met_guess    v
  tv      127      0     met_guess    tv
  q       127      1     met_guess    q
  ps        1      0     met_guess    prse
 ::
```
back into `global_anavinfo.l127.txt`.   We want to retain the efsoi section in `develop`.

Once this is done, we can merge this PR into `develop`.

**Review by RussTreadon-NOAA — APPROVED** [review]
Looks good to me.  

Approve

**Pull request merged**
